ansible/inventories/development

```diff
@@ -1,5 +1,5 @@
 [webworkers]
-192.168.33.15
```

Comment:
@snopoke I think you want to add a file in https://github.com/dimagi/commcarehq-ansible/tree/master/ansible/group_vars where we define this variable.  That way it will get picked up for all environments, alternatively this might be the right place to define it if we don't want to install newrelic on staging/india/development.  @dannyroberts will know better than me.